### DISCUSSION TITLE
Please, disable the "Please star the repo, follow us" banner…

### DISCUSSION THREAD

**Discussion by keul** [damms005/devdb-vscode#47] [Ideas]
…or, at least, add a "never show again" button.

I use many VS code extension, but I never found a so-noisy ones. You are building a very nice tool, but this is too much IMHO. Also I don't see a way to stop this. Am I forced to star your repo (which BTW, I already did)

**Comment by damms005** [comment]
@keul  You really have earned the "unstar" badge lol.

I definitely planned for this feature to *not* be so annoying haha, which is why I allowed disabling it from the settings.

I apologize for this. Darn I even added nice icons to make it less annoying lol.

Anyways, you can already disable it from settings

![image](https://github.com/damms005/devdb-vscode/assets/9839355/3c045655-a6d1-4ac6-93a5-cc9ad9a173c9)


And thanks for the compliments. Glad you find it useful. You'll love the features I am currently working on for v2 more :crossed_fingers:

**Reply by keul** [reply]
ah, my fault then!
I didn't get that this was related to a "new version". On that popup STAR and FOLLOW where more prominent.

**Reply by damms005** [reply]
Yeah it's subtle to notice, but it only pops up on new installation *or* when a new update is installed 💅
